nextflow.config

```diff
@@ -80,64 +85,93 @@ try {
 // }
 
 
-
 profiles {
-    debug { process.beforeScript = 'echo $HOSTNAME' }
+    debug {
+        dumpHashes             = true
+        process.beforeScript   = 'echo $HOSTNAME'
+        cleanup = false
+    }
     conda {
-        params.enable_conda    = true
+        conda.enabled          = true
         docker.enabled         = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     mamba {
-        params.enable_conda    = true
+        conda.enabled          = true
         conda.useMamba         = true
         docker.enabled         = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
         docker.enabled         = true
+        docker.registry        = 'quay.io'
         docker.userEmulation   = true
+        conda.enabled          = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
+    }
+    arm {
+        docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
     }
     singularity {
         singularity.enabled    = true
         singularity.autoMounts = true
+        conda.enabled          = false
         docker.enabled         = false
         podman.enabled         = false
         shifter.enabled        = false
+        apptainer.enabled      = false
         charliecloud.enabled   = false
         singularity.envWhitelist = "LSF_BINDIR,LSF_SERVERDIR,LSF_LIBDIR,LSF_ENVDIR"
         singularity.runOptions = "-B ${projectDir}/assets/martian.lsf.template:/opt/longranger-2.2.2/martian-cs/2.3.2/jobmanagers/lsf.template -B /software -B /etc/lsf.conf --env APPEND_PATH=$LSF_BINDIR:$LSF_SERVERDIR:/software/singularity-v3.9.0/bin"
```

Comment:
I don't think this is the right place for this. Let's talk about this.

Reply:
https://github.com/sanger-tol/genomeassembly/pull/19